tests/supervisors/test_reload.py

```diff
@@ -232,7 +253,18 @@ def test_should_not_reload_when_only_subdirectory_is_watched(
 
         reloader.shutdown()
 
-    @pytest.mark.parametrize("reloader_class", [WatchFilesReload, WatchGodReload])
+    @pytest.mark.parametrize(
+        "reloader_class",
+        [
+            pytest.param(
+                WatchFilesReload,
+                marks=pytest.mark.skipif(
+                    sys.platform == "darwin", reason="Flaky on MacOS"
```

Comment:
Don't you want to narrow it down to only Apple Silicon architecture?
Maybe use platform module for that?

Reply:
How do I do that?

Reply:
I think something along this to check both OS and CPU architecture:

```py
import platform

platform.system() -> "Darwin"
platform.processor() -> "arm"
```

Reply:
done